src/renderer/services/neo.js

```diff
@@ -653,19 +656,31 @@ export default {
   fetchNEP5Balance(address, assetId) {
     const currentNetwork = network.getSelectedNetwork();
 
-    return new Promise((resolve, reject) => {
+    return new Promise((resolve) => {
       return api.nep5.getToken(currentNetwork.rpc, assetId, address)
-        .then((token) => {
+        .then(() => {
           resolve({
-            name: token.name,
```

Comment:
Why does this method no longer return the response containing the token information that is returned from the call? Am I missing something?

Reply:
Good catch, left some test code there by mistake